examples/posenet/src/PoseNet.js

```diff
@@ -1,12 +1,13 @@
 class PoseNet{
-  constructor(modelArch, version, outputStride, inputShape, type, cacheMap, backend, prefer) {
+  constructor(modelArch, version, dilated, outputStride, inputShape, type, cacheMap, backend, prefer) {
     this._modelArch = modelArch;
     this._model = null;
     this._compilation;
     this._execution;
     this._tensorIds = [];
     this._operandIndex = 0;
     this._version = version;
+    this._dilated = dilated;
```

Comment:
For clarity, I would suggest to change variable name to `_use_atrous_conv`. Does it sound?

Reply:
Looks easier to understand! Thanks!